modules/n1ql/pages/n1ql-language-reference/join.adoc

```diff
@@ -1078,31 +1075,31 @@ LIMIT 4;
 ----
 [
   {
-    "airline": "VX",
-    "callsign": "REDWOOD",
-    "destinationairport": "SAN",
-    "name": "Virgin America",
+    "airline": "B6",
+    "callsign": "JETBLUE",
+    "destinationairport": "AUS",
+    "name": "JetBlue Airways",
```

Comment:
again, it's slightly confusing to see data changes in a docs commit.

If this is just an artefact of not having ORDER, would it be worth adding an explicit ordering?
e.g. to at least minimize data changes in future?

(Obviously if the example is clearer without ORDER, then fair enough, is all trade-offs)

Reply:
This is one where (i) the original query gives different results in the new release, and (ii) the updated query gives different results again — but they're consistent each time I run it. So something's obviously changed in the query engine. Another good reason for automated testing!

The point about reproducibility of non-deterministic results is a good one. If an example isn't guaranteed to return the same results each time we should probably have a note explaining that. Or as you say add an ORDER BY clause to guarantee the same results.